sdk/samples/iot/README.md

```diff
@@ -196,9 +202,9 @@ The following samples use x509 authentication to connect to Azure IoT Hub or Azu
 
```

Comment:
Section starting at line 193 is only for Windows CMD.  Needs section for PowerShell.

Reply:
The PR for powershell hasn't been included yet. This will be updated with the powershell update.